packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -1,26 +1,66 @@
 export enum CacheRole {
   None = 'none',
   ReadWrite = 'readwrite',
+  ReadOnly = 'readonly',
 }
 
+export class Any {}
```

Comment:
Specifically looking for approvals on names and shapes in this file today.

Reply:
My apologies. I thought this was the client-protos PR!